src/Hedgehog/Hedgehog.fsproj

```diff
@@ -6,7 +6,7 @@
     <Description>Release with confidence.</Description>
     <Authors>Jacob Stanley;Nikos Baxevanis</Authors>
     <Copyright>Copyright © 2017</Copyright>
-    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\Hedgehog.XML</DocumentationFile>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
Very nice. This looks like a [modern alternative to the old school one](https://docs.microsoft.com/en-us/dotnet/csharp/codedoc). :100: